release/create_release_archives.sh

```diff
@@ -170,6 +170,11 @@ rm -rf "${COMMON_FILES_DIR}/install/kubernetes/helm/istio/test-values/"
 
 ls -l  "${COMMON_FILES_DIR}/install/kubernetes/helm/istio"
 
+TEMP_DIR=$(mktemp -d)
+pushd "${TEMP_DIR}"
+  git clone -b "${CB_BRANCH}" https://github.com/istio/cni.git
+  cp -r cni/deployments/kubernetes/install/helm/istio-cni "${COMMON_FILES_DIR}/install/kubernetes/helm"
```

Comment:
If we do it this way we will need to update https://istio.io/docs/setup/kubernetes/additional-setup/cni/#installation as well.

Not sure if we intentionally made it behave the old way (a tgz file of it), but this seems better to me so we should probably do this and update the docs?

@tiswanso @john-a-joyce @rlenglet @sdake any opinions?

Reply:
This is fine - and changing the documentation is straightforward.  I do think we need to add some something to the README.md as well to indicate the source of cni as currently its magically produced from another repo...

Reply:
@howardjohn re updating the docs, see issue here for cni updates: https://github.com/istio/istio.io/issues/4327

Cheers
-steve